src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { useMemo, useRef, useCallback, useState } from 'react';
+import { Text, TouchableOpacity, StyleSheet, View } from 'react-native';
+import colors from '@styles/colors';
+import BottomSheet, { BottomSheetScrollView } from '@gorhom/bottom-sheet';
+import { GestureHandlerRootView } from 'react-native-gesture-handler';
+import { WeatherComponent } from '@components/WeatherComponent';
+import { ExtraInfoType } from '@components/ExtraInfo';
+import { WarningType } from '@components/Warning';
+import { WeatherType } from '@components/WeatherIcon';
+import { MAIN_HORIZONTAL_OFFSET } from '@styles/constants';
+import { BottomSheetButtons } from '@utils/constants';
+
+export const Sheet = () => {
+    const sheetRef = useRef<BottomSheet>(null);
+
+    const snapPoints = useMemo(() => ['29%', '96%'], []);
+
+    const handleSheetChanges = useCallback((index: number) => {
+        console.log('handleSheetChanges', index);
+    }, []);
+
+    const [sectionWeather, setSectionWeather] = useState(true);
+    const [sectionСlothes, setSectionСlothes] = useState(false);
+
+    const switchWeather = () => {
+        if (sectionWeather) {
+            setSectionWeather(false);
+        } else {
+            setSectionWeather(true);
+            setSectionСlothes(false);
+        }
+        if (sectionWeather === true) {
+            setSectionWeather(true);
+        }
+    };
+
+    const switchСlothes = () => {
+        if (sectionСlothes) {
+            setSectionСlothes(false);
+        } else {
+            setSectionСlothes(true);
+            setSectionWeather(false);
+        }
+        if (sectionСlothes === true) {
+            setSectionСlothes(true);
+        }
+    };
+
+    if (sectionWeather) {
+        return (
+            <View style={styles.container}>
+                <GestureHandlerRootView style={styles.gestureHandlerContainer}>
```

Comment:
![image](https://user-images.githubusercontent.com/52620425/227716326-2b18d806-554e-4215-9604-a0c2c0fe4e6f.png)

You should wrap all elements in <ins>App.tsx</ins>, not every bottom sheet itself